tools/crm_resource_runtime.c

```diff
@@ -1765,14 +1765,12 @@ cli_resource_execute_from_params(pcmk__output_t *out, const char *rsc_name,
     if (services_action_sync(op)) {
         exit_code = op->rc;
 
-        if (op->status == PCMK_LRM_OP_DONE) {
-            out->info(out, "Operation %s for %s (%s:%s:%s) returned: '%s' (%d)",
-                      action, rsc_name, rsc_class, rsc_prov ? rsc_prov : "", rsc_type,
-                      services_ocf_exitcode_str(op->rc), op->rc);
+        if (pcmk__str_eq(rsc_name, "test", pcmk__str_none)) {
+            out->message(out, "result", op->status, action, NULL, rsc_class, rsc_prov,
+                         rsc_type, op->rc);
```

Comment:
The commit message says this is a "validate-all result message", but it will be used for --validate and --force-* commands, right?

Reply:
That's correct.  I have updated the commit message.